### ISSUE TITLE
Chat not logged when viewing multiplayer panel

### ISSUE THREAD

**Issue by Zowdie** [treeform/istrolid-bugs#11]
The chat in a given server is not logged when the player checks the multiplayer section (general chat + servers + playerlist) of the game; any chat that goes on in the original server the player was in is not displayed if the player returns to the server. The game should still be showing the chat since the player never leaves the server.

**Comment by treeform** [comment]
Yes at that time it switches to global chat. Right now you can only be subscribed to one chat channel at a time.
